client/landing/stepper/declarative-flow/internals/steps.tsx

```diff
@@ -65,11 +65,6 @@ export const STEPS = {
 
 	ERROR: { slug: 'error', asyncComponent: () => import( './steps-repository/error-step' ) },
 
-	MIGRATION_ERROR: {
-		slug: 'error',
```

Comment:
Nice to see a duplicate slug step gone! (check `ERROR` step above).

Reply:
Hell yeah!